### DISCUSSION TITLE
Clickhouse networks setting suggestion.

### DISCUSSION THREAD

**Discussion by optimuscream** [akvorado/akvorado#96] [Q&A]
Hi.

Can I use multiple network instance in clickhouse definition in akvorado.yaml like examples below :

```
  name: ipv4-customers
    networks: [192.168.2.0/24, 192.168.3.0/24, 172.10.10.0/24]
    role: customers
  name: servers
    networks: [10.10.10.0/24, 172.16.10.0, 192.168.10.0/24]
    role: servers
```

instead of :

```
  networks:
    192.0.2.0/24:
      name: ipv4-customers
      role: customers
    203.0.113.0/24:
      name: ipv4-servers
      role: servers
    2a01:db8:cafe:1::/64:
      name: ipv6-customers
      role: customers
    2a01:db8:cafe:2::/64:
      name: ipv6-servers
      role: servers
```

If not possible , just my suggestion.
Appreciate for your work.

Best regards.

Thank you.

**Comment by vincentbernat** [comment]
Not a bad idea, but this is not trivial. The current structure is abstracted and reused for other parts of the configuration (for example for SNMP communities). But maybe I'll do that. It's not hard to allow both ways (a list and a map).
